packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -1029,6 +1030,9 @@ class PopupMenuButton<T> extends StatefulWidget {
   /// Theme.of(context).cardColor is used.
   final Color? color;
 
+  /// Change the default icon size.
+  final double iconSize;
```

Comment:
For the sake of consistency with components that have themes, and because PopupMenuButton might have a theme eventually, this should be `double?` and the default should be included with a `widget.iconSize ?? 24.0` epxression.